client/src/pages/useCase/steps/StepCredential.tsx

```diff
@@ -76,6 +79,8 @@ export const StepCredential: React.FC<Props> = ({ step, connectionId, issueCrede
     !credentialsAccepted ? 1000 : null
   )
 
+  protocolVersion = useCredentials().protocolVersion
```

Comment:
What does this do?

Reply:
I'm basically making another call of the function on the off chance that the value changes before the second call to issueCredential. i don't think its doing much, but I added because I wasn't sure if the state value could change while the component was being rendered.